cfgov/tccp/jinja2/tccp/includes/card_list.html

```diff
@@ -74,7 +74,11 @@ <h2>Cards you're looking for have these features:</h2>
                     <h3 class="m-card__heading">{{ card.institution_name }}</h3>
                     <p class="m-card__subtitle">{{ card.product_name }}</p>
                 </div>
-                {{ apr_rating(card.purchase_apr_for_tier_rating) }}
+                <div class="o-apr-rating-group">
```

Comment:
Putting this in o-apr-rating-group applies [this rule](https://github.com/cfpb/consumerfinance.gov/blob/8855e0e8763c9e9e955c7e4de38c304f81b2737a/cfgov/unprocessed/apps/tccp/css/main.less#L204-L206) which justifies the symbols to the right, which we don't want in the card:

<img width="824" alt="image" src="https://github.com/cfpb/consumerfinance.gov/assets/654645/2f938de1-88a3-4a6d-a235-5daf59c3054f">

Is there a way to justify right if we have only one of these? I played around with `:only-of-type` but that doesn't quite fit our need.

Reply:
To fix this issue and also improve the semantics, I ditched the `o-apr-rating-group` class because they're technically not a group if there's only one rating.